Origin: upstream https://github.com/varnishcache/varnish-cache/commit/1d767f469957c7146a59ac8e836437384cb5a2e5
Description: Report varnish version on startup. The version provided by upstream in
70b5a727e18c4ea5c8a9d7cab6b7c68ff2325537 adds the functionality for varnish 6.1 - 6.4
and the one in 23ab35c25f3d7be97d8a386e6fb4ccc5d9ad3ca2 for varnish 6.0
so this patch can be safely removed.
--- a/bin/varnishd/mgt/mgt_main.c	2019-06-05 18:40:14.664780510 +0200
+++ b/bin/varnishd/mgt/mgt_main.c	2019-06-05 18:40:26.100788622 +0200
@@ -849,6 +849,7 @@
 
 	assert(pfh == NULL || !VPF_Write(pfh));
 
+	MGT_Complain(C_DEBUG, "Version: %s", VCS_version);
 	MGT_Complain(C_DEBUG, "Platform: %s", VSB_data(vident) + 1);
 
 	if (d_flag)
